lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3717,7 +3847,9 @@ def get_email_content_response(self, num_emails, task_history_request, with_fail
         self.setup_fake_email_info(num_emails, with_failures)
         task_history_request.return_value = list(self.tasks.values())
         url = reverse('list_email_content', kwargs={'course_id': str(self.course.id)})
-        with patch('lms.djangoapps.instructor.views.api.CourseEmail.objects.get') as mock_email_info:
+        with patch(
```

Comment:
I removed the import of the `CourseEmail` model from the `.../views/api.py` file (now that the code is using the exposed Python API function from the `bulk_email` app). This broke these tests. I was pretty confused for awhile and I was confused how this worked, I think the patch was pointing to the wrong place anyway.